source/clear-linux/tutorials/web-server-install/web-server-install.rst

```diff
@@ -1,106 +1,92 @@
 
-Before installing any new packages, update the |CL| OS with the
-console command:
+Before you install new packages, update the |CL| OS with the following console command:
 
-.. code-block:: console
+.. code-block:: bash
 
    sudo swupd update
 
-Creating a LAMP Server
+Create a LAMP Server
 **********************
 
-A server with Linux\*, Apache\*, MySQL\*, and PHP\* installed is known as a
-LAMP server, allows you to set up a fully functional web server, and enables
-you to host your own website. This tutorial walks you through the process of
-creating a LAMP server by installing and configuring each component. In place
-of MySQL, we will be installing MariaDB which is a drop- in replacement for
-MySQL. Once the LAMP server component installations are complete, we add
-phpMyAdmin to manage your MariaDB databases.
+A LAMP server uses Linux\*, Apache\*, MySQL\*, and PHP\* to set up a fully functional web server and host a website. Note that this tutorial installs MariaDB, which is a drop-in replacement for MySQL. 
+
+This tutorial follows these steps:
 
-Installing Apache
+1. Install Apache.
+2. Change the default configuration and data directory.
+3. Install PHP.
+4. Install MariaDB.
+5. Install phpMyAdmin.
+6. Set up a WordPress database.
+
+Install Apache
 *****************
 
-Apache is an open source HTTP web server application. It can run on several
-operating systems, including |CL|. Alternatively, you could install
-NGINX but this tutorial focuses on implementing an Apache server.
-Go to https://httpd.apache.org/ to learn more about it.
+Apache is an open source HTTP web server application that can run on several operating systems, including |CL|. Go to https://httpd.apache.org/ for more information.
 
 Install the web-server-basic bundle
 ===================================
 
-The web-server-basic bundle contains the packages needed to install the
-Apache software bundle on |CL|.
+The web-server-basic bundle contains the packages needed to install the Apache software bundle on |CL|.
 
 To install the bundle, enter the following command:
 
-.. code-block:: console
+.. code-block:: bash
 
    sudo swupd bundle-add web-server-basic
 
 
 To start the Apache service, enter the following commands:
 
-.. code-block:: console
+.. code-block:: bash
 
    sudo systemctl enable httpd.service
    sudo systemctl start httpd.service
 
 
-To verify that the Apache server application is running, go to your web
-browser and navigate to: http://localhost
+To verify that the Apache server application is running, open a web browser and navigate to: http://localhost.
 
-If the service is running you will see the message "This web server is
-operational from host." on your browser as shown in figure 1.
+If the service is running, a confirmation message appears, as shown in figure 1.
```

Comment:
This figure is rendering, but it needs a little TLC. In short, revise to: 

`
.. figure:: figures/web-server-install-1.png
   :alt: This web server is operational from host.
   :scale: 50
`
I know, .rst is super fussy. 🤷‍♂️  It's Python-based, so white-space is meaningful. 
What changed? The text `:alt:` and `:scale:` were moved back one space to align with the "f" in figure above it.  See first figure under "Using phpMyAdmin..." for a correct example.

Reply:
Even after these updates, the image still looks back. I have edited the image (removed white space) and it looks better now.